generate_engagement_analysis_files.py

```diff
@@ -0,0 +1,109 @@
+from collections import OrderedDict
+import argparse
+import json
+import csv
+
+from core_data_modules.logging import Logger
+
+from src.lib import PipelineConfiguration
+
+Logger.set_project_name("IMAQAL")
+log = Logger(__name__)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generates engagement data stored in multiple CSV sheets for analysis"
+                                                 " and visualization")
+
+    parser.add_argument("user", help="User launching this program")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
+                        help="Path to the pipeline configuration json file"),
+    parser.add_argument("demog_map_json_input_dir", metavar="demog-map-json-input-dir",
+                        help="Path to a directory to read per episode demog map .json files ")
```

Comment:
I still think you should describe the required naming format here i.e. <rqa_raw_field>_demog_map.json.

Reply:
updated thanks!